translate/lang/data.py

```diff
@@ -102,6 +102,7 @@
     'jbo': ('Lojban', 1, '0'),
     'jv': ('Javanese', 2, '(n != 1)'),
     'ka': ('Georgian', 1, '0'),
+    'kab': ('Kabyle', 1, '(n > 1)'),
```

Comment:
This is wrong. Either the number of plurals is greater than `1` or `(n > 1)` is replaced by `0`.